src/events.cpp

```diff
@@ -8,18 +8,90 @@
 #include "item.h"
 #include "player.h"
 
-Events::Events() : scriptInterface("Event Interface") { scriptInterface.initState(); }
+namespace {
 
-bool Events::load()
+LuaScriptInterface scriptInterface{"Event Interface"};
+
+struct CreatureHandlers
+{
+	int32_t onChangeOutfit = -1;
+	int32_t onAreaCombat = -1;
+	int32_t onTargetCombat = -1;
+	int32_t onHear = -1;
+	int32_t onChangeZone = -1;
+	int32_t onUpdateStorage = -1;
+};
+
+struct PartyHandlers
+{
+	int32_t onJoin = -1;
+	int32_t onLeave = -1;
+	int32_t onDisband = -1;
+	int32_t onShareExperience = -1;
+	int32_t onInvite = -1;
+	int32_t onRevokeInvitation = -1;
+	int32_t onPassLeadership = -1;
+};
+
+struct PlayerHandlers
+{
+	int32_t onBrowseField = -1;
+	int32_t onLook = -1;
+	int32_t onLookInBattleList = -1;
+	int32_t onLookInTrade = -1;
+	int32_t onLookInShop = -1;
+	int32_t onLookInMarket = -1;
+	int32_t onMoveItem = -1;
+	int32_t onItemMoved = -1;
+	int32_t onMoveCreature = -1;
+	int32_t onReportRuleViolation = -1;
+	int32_t onReportBug = -1;
+	int32_t onRotateItem = -1;
+	int32_t onTurn = -1;
+	int32_t onTradeRequest = -1;
+	int32_t onTradeAccept = -1;
+	int32_t onTradeCompleted = -1;
+	int32_t onPodiumRequest = -1;
+	int32_t onPodiumEdit = -1;
+	int32_t onGainExperience = -1;
+	int32_t onLoseExperience = -1;
+	int32_t onGainSkillTries = -1;
+	int32_t onWrapItem = -1;
+	int32_t onInventoryUpdate = -1;
+	int32_t onNetworkMessage = -1;
+	int32_t onSpellCheck = -1;
+};
+
+struct MonsterHandlers
 {
+	int32_t onDropLoot = -1;
+	int32_t onSpawn = -1;
+};
+
+CreatureHandlers creatureHandlers;
```

Comment:
Use `struct { ... } creatureHandlers;` so you create everything in a single statement

Reply:
Thanks